wagtail/admin/templates/wagtailadmin/shared/button.stories.tsx

```diff
@@ -0,0 +1,218 @@
+import React from 'react';
+import { Pattern, generateDocs } from 'storybook-django/src/react';
+
+export default {
+  title: 'Shared / buttons',
+};
+
+export const all = (args) => (
```

Comment:
```suggestion
const Template = ({ url }) => {
```

Let's not export this directly, let's use this as a `Template` similar to other storybook.